feat: prepare map work and temp map img

ermish opened the pull request:
Resolves #146 

this makes it easier to visualize the upcoming welcome banner changes  

# What changed
- [x] set up temp map to better visualize
- [x] created vue map container

# Testing instructions
- a map shows!

ann-kilzer reviewed (commented):
I like that we're getting a clearer vision for the site. Two notes:

1. Let's be careful with any images where we don't have a license, this looks very Google
2. Consider zooming out to all of Japan to avoid a Tokyo bias (though the final product might be most useful on a local scale so perhaps I'm being overly sensitive)

- https://www.openstreetmap.org/#map=5/35.800/129.595
- https://commons.wikimedia.org/w/index.php?search=japan+map&title=Special:MediaSearch&go=Go&type=image

I added a note about Cypress selectors too

ermish commented:
> I like that we're getting a clearer vision for the site. Two notes:
> 
> 1. Let's be careful with any images where we don't have a license, this looks very Google
> 2. Consider zooming out to all of Japan to avoid a Tokyo bias (though the final product might be most useful on a local scale so perhaps I'm being overly sensitive)
> 
> * https://www.openstreetmap.org/#map=5/35.800/129.595
> * https://commons.wikimedia.org/w/index.php?search=japan+map&title=Special:MediaSearch&go=Go&type=image
> 
> I added a note about Cypress selectors too

fair enough! 
I'll switch out the temporary image for an openstreetmap version

ermish commented on cypress/e2e/home.cy.ts:
```diff
@@ -3,6 +3,6 @@ describe('Visits the home page', () => {
     })
     it('Displays the Hero Section', () => {
-        cy.get('h1').contains('Connecting people in Japan to the healthcare services they need, in the languages they need.')
+        cy.get('#search-button').contains('Search')
```
I find it really suprising that they discourage ids. Id's are meant to be static (unless they're generated). Totes agree with never using classes though for an identifier. 

We'll need to add in more thorough tests as this one doesn't really do much yet. 👍 
 I'll create a couple testing github issues when we get a little closer to having a complete page

ermish replied:
updated. using `data-testid` as this is usually the best long term, reliable approach for element access.

theyokohamalife reviewed (approved):
LGTM

ann-kilzer reviewed (changes requested):
Will approve once the map file is non-Google

ermish commented:
> Will approve once the map file is non-Google

@ann-kilzer  please review agin

ermish commented:
> I like that we're getting a clearer vision for the site. Two notes:
> 
> 1. Let's be careful with any images where we don't have a license, this looks very Google
> 2. Consider zooming out to all of Japan to avoid a Tokyo bias (though the final product might be most useful on a local scale so perhaps I'm being overly sensitive)
> 
> * https://www.openstreetmap.org/#map=5/35.800/129.595
> * https://commons.wikimedia.org/w/index.php?search=japan+map&title=Special:MediaSearch&go=Go&type=image
> 
> I added a note about Cypress selectors too

updated
